### ISSUE TITLE
[Bug]: Latest version (1.4.0) does not run on Windows

### ISSUE THREAD

**Issue by enarkay** [DmitriySalnikov/GodotPCKExplorer#13]
### Commit or version

1.4.0

### On which operating systems the error occurs

Windows

### Issue description

The latest version (v1.4.0) of the program (the UI variant) does not run.
My OS is Windows 10 64bit.
I've tested and tried multiple times, even adding the folders as exceptions to windows firewall, but no success.
The previous version (v1.3.3) runs just fine.

### Steps to reproduce

N/A

**Comment by DmitriySalnikov** [comment]
What version of .NET 8 have you installed? And what error appears at startup? Are there any logs in `%appdata%/GodotPCKExplorer`?

You need to install `.NET Desktop Runtime` as I mentioned in the changes, and in README.md.
https://github.com/DmitriySalnikov/GodotPCKExplorer/releases/tag/1.4.0#:~:text=.NET%20Desktop%20Runtime

**Comment by enarkay** [comment]
> What version of .NET 8 have you installed? And what error appears at startup? Are there any logs in `%appdata%/GodotPCKExplorer`?
> 
> You need to install `.NET Desktop Runtime` as I mentioned in the changes, and in README.md. https://github.com/DmitriySalnikov/GodotPCKExplorer/releases/tag/1.4.0#:~:text=.NET%20Desktop%20Runtime

I'm sorry you're right. Installing .NET 8 solved the problem. I initially misunderstood the instructions in such way that the release comes with .NET 8 pre-included. Sorry for my negligence and thank you!

**Comment by DmitriySalnikov** [comment]
@enarkay I investigated the problem a bit further and realized that when you run the UI version, it doesn't create a window asking you to install the correct version of `.NET` as I expected. In 5ae709c45f174f3d14e10cbfdcacada1625f352c I changed this and now the console is created differently, but a normal window with a `Download` button will be displayed.

![vmware_FejK5TWcD9](https://github.com/DmitriySalnikov/GodotPCKExplorer/assets/7782218/1f24b325-2de6-4886-9750-a627374adc5a)
![vmware_16LTn8O9Vl](https://github.com/DmitriySalnikov/GodotPCKExplorer/assets/7782218/7948ce9c-e08d-4de7-95c7-d81602b321af)

I think I won't update the version number, but I will replace the exe of the current version for new users.
